sdk/core/azure-core/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ set(
     inc/azure/core/dll_import_export.hpp
     inc/azure/core/etag.hpp
     inc/azure/core/exception.hpp
+    inc/azure/core/hash.hpp
```

Comment:
Other SDKs have placed these in the Crypto namespace.  Is there a reason not to do the same?

Reply:
I am OK with moving these to an `Azure::Core::Cryptography` sub-namespace. @JeffreyRichter what do you think?

Would that change the file location in anyway?

Reply:
A Cryptography namespace is fine. But, what will be in it? Hash and Md5Hash?
This might be OK because I expect very few customers to actually use these things.

Reply:
Would we move `Azure::Storage::Crc64` to `Azure::Storage::Cryptography::Crc64`, as well?